src/clone.rs

```diff
@@ -94,10 +94,16 @@ macro_rules! to_type_before {
     (@weak $($variable:ident).+ as $rename:ident) => (
         let $rename = $crate::clone::Downgrade::downgrade(&$($variable).+);
     );
-    (@ $keyword:ident $($variable:ident).+ $(as $rename:ident)?) => (
+    (@strong $variable:expr) => (
+        let $variable = $variable.clone();
+    );
+    (@weak $variable:expr) => (
```

Comment:
Can we get tests for these two cases?

Reply:
They can't get called actually, it's only to prevent the macro to fail on "$variable:expr". I can add a comment explaining it though.

Reply:
If it can't be called, why would it be needed then? :) (Yes please add an explanation comment)

Reply:
I added the comment. Can you tell me if it's clear enough please?